### PR TITLE
Readme: Add Windows compile instructions

### PR DESCRIPTION
**Pull request by EwoutH**
Also remove $-signs to make copy-paste easier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,25 @@ The currently planned/on-going integration work includes:
...
-    $ git submodule update --init --recursive
-    $ cmake ./
-    $ make -j
-    $ make -j install
+The basic commands to clone, build, and install brunsli on Linux are:
+
+    git clone https://github.com/google/brunsli.git
+    cd brunsli
+    git submodule update --init --recursive
+    cmake ./
```
Comment (eustas) [inline README.md]: it is better to build in a separate dir e.g. ./out or ./build

File: README.md [hunk 2]
```diff
@@ -27,13 +27,25 @@ The currently planned/on-going integration work includes:
...
+
+### Windows
+
+The basic commands to clone and build brunsli on Windows with Visual Studio are:
+
+    git clone https://github.com/google/brunsli.git
+    cd brunsli
+    git submodule update --init --recursive
+    cmake ./
+    msbuild brunsli.sln
```
Comment (eustas) [inline README.md]: Could separate linux build to "preamble" (git + cmake) and make. Then windows build could be expressed as "preamble" + msbuild

File: README.md (hunk 1 above)
Comment (EwoutH) [reply README.md]: Agreed, I just copied them as they existed

**Comment by EwoutH** [comment]
Resolved comments, and I also added a `--depth=1` flag to git to minimize download and folder sizes.

**Comment by eustas** [comment]
Thanks

**Pull request merged**